jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -72,3 +72,6 @@ OPENJ9_LIBS_SUBDIR      := @OPENJ9_LIBS_SUBDIR@
 
 # Windows
 MSVCP_DLL               := @MSVCP_DLL@
+
+COPY_JVM_CFG_FILE       := true
+
```

Comment:
Please remove this blank line (but not the new-line at the end of the line before it).

Reply:
Already removed.